components/org.wso2.carbon.identity.client.attestation.filter/src/main/java/org/wso2/carbon/identity/client/attestation/filter/ClientAttestationProxy.java

```diff
@@ -292,4 +299,12 @@ private Response buildResponse(String errorDescription, Response.Status status)
 
         return Response.status(status).entity(errorJSON).build();
     }
+
+    private static String removeTrailingSlash(String url) {
```

Comment:
Do we have any purpose for this method as static

Reply:
fixed.